### DISCUSSION TITLE
Where are instrunctions or help for Aris CSS stuff?

### DISCUSSION THREAD

**Discussion by joefiesta** [Aris-t2/CustomCSSforFx#568] [General]
I installed all the stuff I use for Aris's CSS stuff long, long ago.  Now, I need to update but I can't find any instructions for using his stuff.  And, I surely don't remember.  I've looked all around this github topid and find nothing.

where can I find instructions?

**Comment by glennpc** [comment]
https://github.com/Aris-t2/CustomCSSforFx/blob/master/README.md

**Comment by Speravir** [comment]
In addition to @glennpc:

The README is also embedded on the startpage of this collection, see more precisely https://github.com/Aris-t2/CustomCSSforFx#readme

But we can even link directly to the [section for Instructions/Howto/Readme](https://github.com/Aris-t2/CustomCSSforFx#user-content-instructions--howto--readme).

**Comment by Aris-t2** [comment]
I think we can "close" this one, right?
